packages/grafana-data/src/transformations/fieldReducer.test.ts

```diff
@@ -23,288 +23,318 @@ function createField<T>(name: string, values?: T[], type?: FieldType): Field<T>
   };
 }
 
-describe('Stats Calculators', () => {
-  const basicTable = createDataFrame({
-    fields: [
-      { name: 'a', values: [10, 20] },
-      { name: 'b', values: [20, 30] },
-      { name: 'c', values: [30, 40] },
-    ],
-  });
+describe('fieldReducer.ts', () => {
+  describe('Stats Calculators', () => {
+    const basicTable = createDataFrame({
+      fields: [
+        { name: 'a', values: [10, 20] },
+        { name: 'b', values: [20, 30] },
+        { name: 'c', values: [30, 40] },
+      ],
+    });
 
-  it('should load all standard stats', () => {
-    for (const id of Object.keys(ReducerID)) {
-      const reducer = fieldReducers.getIfExists(id);
-      const found = reducer ? reducer.id : '<NOT FOUND>';
-      expect(found).toEqual(id);
-    }
-  });
+    it('should load all standard stats', () => {
+      for (const id of Object.keys(ReducerID)) {
+        const reducer = fieldReducers.getIfExists(id);
+        const found = reducer ? reducer.id : '<NOT FOUND>';
+        expect(found).toEqual(id);
+      }
+    });
 
-  it('should fail to load unknown stats', () => {
-    const names = ['not a stat', ReducerID.max, ReducerID.min, 'also not a stat'];
-    const stats = fieldReducers.list(names);
-    expect(stats.length).toBe(2);
+    it('should fail to load unknown stats', () => {
+      const names = ['not a stat', ReducerID.max, ReducerID.min, 'also not a stat'];
+      const stats = fieldReducers.list(names);
+      expect(stats.length).toBe(2);
 
-    const found = stats.map((v) => v.id);
-    const notFound = difference(names, found);
-    expect(notFound.length).toBe(2);
+      const found = stats.map((v) => v.id);
+      const notFound = difference(names, found);
+      expect(notFound.length).toBe(2);
 
-    expect(notFound[0]).toBe('not a stat');
-  });
-
-  it('should calculate basic stats', () => {
-    const stats = reduceField({
-      field: basicTable.fields[0],
-      reducers: [ReducerID.first, ReducerID.last, ReducerID.mean, ReducerID.count, ReducerID.diffperc, ReducerID.delta],
+      expect(notFound[0]).toBe('not a stat');
     });
 
-    expect(stats.first).toEqual(10);
-    expect(stats.last).toEqual(20);
-    expect(stats.mean).toEqual(15);
-    expect(stats.count).toEqual(2);
-    expect(stats.diffperc).toEqual(100);
-    expect(stats.delta).toEqual(10);
-  });
+    it('should calculate basic stats', () => {
+      const stats = reduceField({
+        field: basicTable.fields[0],
+        reducers: [
+          ReducerID.first,
+          ReducerID.last,
+          ReducerID.mean,
+          ReducerID.count,
+          ReducerID.diffperc,
+          ReducerID.delta,
```

Comment:
Nesting the describe changes this from being on one line to broken out for being too long, not a functionality change.